src/plugins/geomap/geomap-en.hbs

```diff
@@ -1303,3 +1303,53 @@ var wet_boew_geomap = {
 		</details>
 	</section>
 </section>
+
+<section id="aoiZoomOnLoad">
+	<h2>Zoom to AOI on page load</h2>
+
+	<form class="wb-geomap-filter form-inline mrgn-bttm-md" data-bind-to="bc-geomap">
+		<div class="form-group">
+			<label for="zoomBCArea">B.C. Regions:</label>
+			<select class="form-control" data-filter="aoi" id="zoomBCArea">
+				<option value="63, -110, 50, -128" selected>Select a region</option>
```

Comment:
This value uses commas between each points, but others don't; please make it consistent. 

Whether it should be with or without commas though is a decision that should be based on what's currently used for other configs of Geomap and also the Action manager.